### PR TITLE
Ratings and Reviews Basic structure with hard-coded data

### PR DESCRIPTION
**Pull request by jordanabbasi**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import ProductContext from '../context';
-import Overview from './Overview/Overview';
-import QA from './Q&A/QA';
+import Overview from './Overview/Overview.jsx';
+import QA from './Q&A/QA.jsx';
+import RR from './R&R/RR.jsx';
```
Comment (alishaedington) [inline client/src/components/App.jsx]: remove file extensions

File: client/src/components/R&R/RR.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/extensions */
+import React from 'react';
+import Breakdown from './Breakdown.jsx';
+import ReviewsList from './ReviewsList.jsx';
+
+export default function RR() {
+  return (
+    <div className="rr-container">
+      <h2 className="rr-header">Ratings & Reviews</h2>
```
Comment (alishaedington) [inline client/src/components/R&R/RR.jsx]: make header <a> so I can link to it from top of page

File: client/src/components/App.jsx (hunk 1 above)
Comment (jordanabbasi) [reply client/src/components/App.jsx]: Ahh good catch!

File: client/src/components/R&R/StarFilter.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+export default function StarFilter({stars}) {
+  return (
+    <div className="rr-star-filter">
+      <div className="rr-star-filter-child rr-n-stars">
+        {`${stars} star${stars > 1 ? 's' : ' '}`}
+      </div>
+      <ProgressBar percent={50} />
+      <div className="rr-star-filter-child">
+        500
```
Comment (alishaedington) [inline client/src/components/R&R/StarFilter.jsx:12]: what is the 500 for? will it be the amount of ratings for that amount of stars?

**Review by alishaedington — DISMISSED** [review]
Looks really good! great job!

File: client/src/components/R&R/RR.jsx (hunk 2 above)
Comment (alishaedington) [reply client/src/components/R&R/RR.jsx]: make header an anchor 'a' element*

File: client/src/components/R&R/StarFilter.jsx (hunk 3 above)
Comment (jordanabbasi) [reply client/src/components/R&R/StarFilter.jsx:12]: Yes.

**Review by alishaedington — APPROVED** [review]
good job

**Pull request merged**